src/main/java/LottoGameApplication.java

```diff
@@ -0,0 +1,15 @@
+import controller.LottoGameController;
+import domain.LottoGenerator;
+import domain.RandomLottoGenerator;
+import view.LottoGameView;
+
+public class LottoGameApplication {
+	public static void main(String[] args) {
+		LottoGameView lottoGameView = new LottoGameView();
+		LottoGenerator lottoGenerator = new RandomLottoGenerator();
+		LottoGameController lottoGameController
+			= new LottoGameController(lottoGameView, lottoGenerator);
```

Comment:
줄 바꿈까지는 필요없는 길이인 것 같아요. 🙂

```suggestion
		LottoGameController lottoGameController = new LottoGameController(lottoGameView, lottoGenerator);
```